shape_msgs/msg/Bounds.msg

```diff
@@ -0,0 +1,25 @@
+# Bounds which are useful in several contexts, e.g. to avoid spawning with other object overlap,
+# or parking in a spot that is too small.
+# Certain goals or points might be valid for a small object, but not suitable for a large one,
+# or a differently shaped one.
+# Bounds can be also checked to ensure certain scenario conditions are met.
+# For entities, these limits are relative to entity's canonical link transform, following ROS rep-103 convention.
+
+# As bounds are optional in most interfaces, TYPE_EMPTY signifies empty bounds, to be understood as "unbounded".
+# Otherwise, the fields are expected to define a valid volume.
+# For spawning with a named pose, you should check whether the entity simulation model fits within the bounds
+# before calling SpawnEntity, to avoid overlaps and unstable behavior.
+
+# bounds type
+uint8 TYPE_EMPTY         = 0      # No bounds. The points vector will be empty.
+uint8 TYPE_BOX           = 1      # Axis-aligned bounding box, points field should have two values,
```

Comment:
What frame is the axis aligned bounding box aligned to?

Reply:
That's an entirely reasonable question, and looking back, there is no frame specification anywhere. @adamdbrw was there a conversation about this in https://github.com/ros-simulation/simulation_interfaces/pull/1

Reply:
It is specified:
For entities, these limits are relative to entity's canonical link transform, following ROS rep-103 convention.